Sources/StreamChatUI/Utils/String+Extensions_Tests.swift

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+@testable import StreamChatUI
+import XCTest
+
+class String_Extensions_Tests: XCTestCase {
```

Comment:
nit: a true positive case for `containsOnlyEmoji` is not covered

Reply:
Thanks for this! Added test for `containsOnlyEmoji`